modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/utils/OAuth2Constant.java

```diff
@@ -50,6 +50,8 @@
     public static final String UNSUPPORTED_GRANT_TYPE = "unsupported_grant_type";
 
     public final static String CALLBACK_URL = "http://localhost:8490/playground2/oauth2client";
+    public final static String CALLBACK_URL_REGEXP = "regexp=http:\\/\\/localhost:8080\\/playground2\\/oauth2client[\\?]?((\\w+)=(\\w+)&?)+";
```

Comment:
for integration tests, we are using 8490 for tomcat, to avoid port conflicts with other products. Have to use same port here also.